packages/ui-default/templates/partials/contest_user.html

```
@@ -13,6 +13,9 @@
       {% if tsdoc.endAt and tsdoc.endAt < tdoc.endAt and tdoc.endAt.getTime() > Date.now() and (not tdoc.duration or not tsdoc.startAt or tsdoc.startAt.getTime() + tdoc.duration * 3600000 > Date.now()) %}
       | <a name="resume_contest" data-uid="{{ tsdoc.uid }}">{{ _('Resume') }}</a>
       {% endif %}
+      {% if tdoc.beginAt.getTime() > Date.now() %}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)